blocks/url-input/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import './style.scss';
+import { __ } from 'i18n';
+import { Component } from 'element';
+import { IconButton } from 'components';
+
+class UrlInput extends Component {
+	constructor() {
+		super( ...arguments );
+		this.toggle = this.toggle.bind( this );
+		this.submitLink = this.submitLink.bind( this );
+		this.state = {
+			expanded: false,
+		};
+	}
+
+	toggle() {
+		this.setState( { expanded: ! this.state.expanded } );
+	}
+
+	submitLink( event ) {
+		event.preventDefault();
+		this.toggle();
+	}
+
+	render() {
+		const { url, onChange } = this.props;
```

Comment:
Wondering if `url` should be managed in state, and the parent component instead binds to `onSubmit`.

Maybe also, should `url` be reset when toggling the expanded state?

My thinking is "url" is state relevant only so long as the input is expanded.